### PR TITLE
Update fs docs and readme

### PR DESCRIPTION
**Pull request by DevinHillenius**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -108,25 +108,6 @@ for [emacs](https://github.com/CodeGra-de/CodeGra.el),
 [vim](https://github.com/CodeGra-de/CodeGra.vim) and more are being created
 upon request.
 
-## Privacy
```
Comment (olmokramer) [inline README.md]: Why has this section been removed?

File: README.md [hunk 2]
```diff
@@ -108,25 +108,6 @@ for [emacs](https://github.com/CodeGra-de/CodeGra.el),
...
-You can use CodeGra.fs for any CodeGrade instance. The application does a
-version check at every startup, this is done by doing a request to
-`https://codegra.de/.cgfs.json` and to `https://codegra.de/.cgfs.version`. We do
-not collect any personal information at these routes. It is currently not
-possible to disable this version check.
-
-## Support
```
Comment (olmokramer) [inline README.md]: Why has this section been removed?

File: docs/changelog.rst [hunk 3]
```diff
@@ -1,10 +1,28 @@
 Changelog
 ==========
 
+Version 1.0.0
```
Comment (olmokramer) [inline docs/changelog.rst]: We're about to release v1.1.1, so maybe we should add a  section for v1.1.0 and v1.1.1

File: README.md (hunk 2 above)
Comment (DevinHillenius) [reply README.md]: huh, fixed

File: README.md (hunk 1 above)
Comment (DevinHillenius) [reply README.md]: huh, weird, fixed

**Pull request merged**